api/rds.proto

```diff
@@ -308,6 +309,9 @@ message Route {
     RouteAction route = 2;
     // Return a 302 redirect.
     RedirectAction redirect = 3;
+    // Return a 401/403 when auth checks fail.
+    // [V2-API-DIFF] new in v2.
+    AuthAction auth = 6;
```

Comment:
yeah per @htuch this should move out of the oneof and be its own thing. Same for virtual host.

Reply:
done